src/components/Company/DocumentSignerFlow/DocumentSignerFlow.tsx

```diff
@@ -16,7 +17,9 @@ export const DocumentSignerFlow = ({
   companyId,
   signatoryId,
   onEvent,
+  dictionary,
 }: DocumentSignerFlowProps) => {
+  useComponentDictionary('Company.DocumentList', dictionary)
```

Comment:
On this one, shouldn't this be moved to the company DocumentSigner component itself? We're also missing this override for the company SignatureForm component with `Company.SignatureForm`

Thinking we probably need to figure out a way to set these at the flow level 🤔 for the purposes of this PR wondering if it's sufficient to just set them on the block level components and we can revisit setting them on flows separately?

Alternatively we may need to refactor to consolidate translations into a single file if that's the direction we want to go